### PR TITLE
Add comments for test container dockerfile

### PR DESCRIPTION
**Pull request by thom-at-redhat**
# Description

Add comments for each line in test container Dockerfile

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I run `go fmt ./...` to check that my code is properly formatted
- [ ] I run `go vet ./...` to check that my code is free of common Go style mistakes
- [ ] I run `make lint` to prints out coding style mistakes and fix them

### CODE REVIEW

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
be3f49fe-df10-4cd4-bf71-90685c90532c
fee87f88-5516-48f7-b560-25fc0de5dcad

File: test-container/Dockerfile [hunk 1]
```diff
@@ -1,53 +1,84 @@
...
+###### Stage 1: Build the FDO package ######
+############################################
+# Use full ubi8 as builder environment
 FROM registry.access.redhat.com/ubi8/ubi AS builder
+# Download specific version of staticcheck
 RUN curl -L https://github.com/dominikh/go-tools/releases/download/2021.1.2/staticcheck_linux_amd64.tar.gz | tar xz
+# Download specific version of gosec
 RUN curl -L https://github.com/securego/gosec/releases/download/v2.9.5/gosec_2.9.5_linux_amd64.tar.gz | tar xz
 
+# Install RHEL 8 utilties to /mnt/rootfs
```
Comment (loadtheaccumulator) [inline test-container/Dockerfile:13]: typo "utilties"

**Review by loadtheaccumulator — COMMENTED** [review]
I stopped about half way through with the question, does every line have to be commented? Some of the steps are common to Dockerfiles.

File: test-container/Dockerfile (hunk 1 above)
Comment (thom-at-redhat) [reply test-container/Dockerfile]: Thank you, fixed

**Pull request merged**